python_backend/Dockerfile

```diff
@@ -4,6 +4,8 @@ COPY ./triton_client /triton_client
 WORKDIR /triton_client
 
 RUN pip install --no-cache-dir -r requirements.txt
+# Dependency for OpenCV
+RUN apt-get update && apt-get install -y libgl1 
```

Comment:
A quick google search told me that maybe we should put the RUN apt-get ahead of the pip install -- lemme know if this makes sense

https://stackoverflow.com/questions/55313610/importerror-libgl-so-1-cannot-open-shared-object-file-no-such-file-or-directo